src/main/java/org/resitrack/controller/PersonController.java

```diff
@@ -51,13 +94,76 @@ public void searchPeopleByName() {
         }
     }
 
+    /**
+     * This function is used to delete one or many people
+     *
+     * @param name
```

Comment:
please explain where is this param come from, what is value of it

Reply:
Đã update